modules/build/src/main/scala/scala/build/input/Inputs.scala

```diff
@@ -304,17 +311,27 @@ object Inputs {
         else
           Left(s"""$arg does not contain shebang header
                   |possible fixes:
-                  |  Add '#!/usr/bin/env scala-cli shebang' to the top of the file
+                  |  Add '#!/usr/bin/env $fullProgramCall' to the top of the file
```

Comment:
we should always advise to use the `shebang` sub-command in the `shebang` header, I think
```suggestion
                  |  Add '#!/usr/bin/env ${programInvokeData.progName} shebang' to the top of the file
```

Reply:
In this If branch it's known that shebang command has been called first, that's why it's full call name here.

Reply:
Ah yeah... got confused, y're right